templates/instanceha/bin/instanceha.py

```diff
@@ -419,7 +419,11 @@ def _check_kdump(stale_services):
         #logging.debug("address is %s" % address[0])
 
         # short hostname
-        name = socket.gethostbyaddr(address[0])[0].split('.', 1)[0]
+        try:
+            name = socket.gethostbyaddr(address[0])[0].split('.', 1)[0]
+        except Exception as msg:
```

Comment:
It would be better to catch something more specific than general exception.

Reply:
With the generic exception we will catch:
ERROR:root:Failed reverse dns lookup for: 8.8.8.88 - [Errno 1] Unknown host (valid dns server, wrong or unknown ip)
ERROR:root:Failed reverse dns lookup for: 8.8.8.88 - [Errno 2] Host name lookup failure (dns server unreachable or bogus response)
that is the same returned by except socket.herror or similar afaik.